### ISSUE TITLE
installation fails with class not found

### ISSUE THREAD

**Issue by luizkowalski** [hschne/rails-mini-profiler#4]
### Issue Summary

I just add this gem on a project and the installation failed right out of bat, saying that `RailsMiniProfiler::Storage::Memory` does not exist. In fact, I couldn't find this class in the project.

### Steps to Reproduce

Just add the gem to Gemfile and run the installation command 

### Technical details

* Rails 6.1.4
* Ruby 3.0.2

**Comment by hschne** [comment]
Fixed in #5. 

You can fix up the initializer by updating `config.storage = RailsMiniProfiler::Storage::Memory`
to `config.storage = RailsMiniProfiler::Storage` in `lib/generators/rails_mini_profiler/templates/rails_mini_profiler.rb.erb` :) 

Let me know how it goes :bow:

**Comment by luizkowalski** [comment]
The installation went through but now I have a different error:

```
2790459895 is out of range for ActiveModel::Type::Integer with limit 4 bytes
```

I believe it is because I don't use sequential ids but UUIDs
